scipy/interpolate/_ppoly.pyx

```diff
@@ -247,7 +247,7 @@ def integrate(double_or_complex[:,:,::1] c,
 @cython.wraparound(False)
 @cython.boundscheck(False)
 @cython.cdivision(True)
-def real_roots(double[:,:,::1] c, double[::1] x, int report_discont,
+def real_roots(double[:,:,::1] c, double[::1] x, double y, int report_discont,
                int extrapolate):
```

Comment:
Same as int in C, but cython does automatic conversions to/from python booleans when necessary.